### DISCUSSION TITLE
SOC dashboards not showing events until modal is closed & reopenned

### DISCUSSION THREAD

**Discussion by sempervictus** [Security-Onion-Solutions/securityonion#8427] [Unsupported Versions] [answered]
Whenever i bring up a dashboard or query in the SOC UI, the actual events are not displayed:
![image](https://user-images.githubusercontent.com/1331084/182150862-2f212f6c-5bd0-41a4-b054-634936b81b1d.png)
If i collapse and expand the Events modal, it will _sometimes_ show events but clearly missing data compared to the Kibana interface.
Concurrently, column selections set for events do not persist forcing analysts to rebuild their views after every query, so even _if_ an event is shown it doesn't present the columns required for rapid assay.

This doesnt strike me as user-error given the intermittent behavior, but given the number of moving parts involved, i wouldn't be _too_ surprised if that turns out to be the case. Has anyone run into these SOC UI issues and have any ideas about how they can be remedied?

**Comment by dougburks** [comment, marked as answer]
I don't believe anybody else has reported anything like this.

What browser are you using? From https://docs.securityonion.net/en/2.3/soc.html:
> We recommend chromium or chromium-based browsers such as Google Chrome. Other browsers may work, but fully updated chromium-based browsers provide the best compatibility.

Have you tried accessing SOC in an incognito window?

Have you tried accessing SOC from another browser with no plugins?

Have you tried accessing SOC from another computer?

Have you tried opening the browser console and looking for additional clues?

**Reply by dougburks** [reply]
We just released a new video on SOC Dashboards that shows how SOC Dashboards operates under normal conditions:
https://youtu.be/xUBhyF7se8s

**Comment by sempervictus** [comment]
Thanks for the sanity check - good to know its just local gremlins.
I'm seeing this in Linux Firefox current (Archlinux), with no-script trusting the site and with all plugins disabled in incognito.
After blowing away the cache on Chrome-current, it looks like _that_ is working correctly. So if this is a niche linux firefox issue, then it'll probably get fixed or more broken the next time they do a version bump. Thanks
